### DISCUSSION TITLE
Allow money sprites to animate during pause mode

### DISCUSSION THREAD

**Discussion by tellovishous** [OpenRCT2/OpenRCT2#17036] [Feature requests] [answered]
I would like it if the floating money sprites could still animate when the game is paused, mainly for the "Allow building in pause mode" cheat. When the cheat is enabled, any money sprites that spawn remain static where they happened, which can be annoying & distracting when many purchases/refunds occur in a localized area (e.g. when building a track in pause mode) and clutter up the view.

**Comment by ocalhoun6** [comment]
If making them animated while the game is paused is too difficult, a slightly worse (but perhaps easier to implement) solution would be to not generate any new floating money sprites while the game is paused, even if money is spent/gained.

**Reply by tellovishous** [reply]
Another option could be that if they don't animate in pause mode, then at least have them disappear after a few seconds like they usually do.

**Comment by tellovishous** [comment, marked as answer]
Looks like this has finally been addressed in #21102, so this can be closed now.
